src/galileo/handlers/openai_agents.py

```diff
@@ -0,0 +1,333 @@
+import logging
+from datetime import datetime
+from typing import Any, Optional
+
+from agents import TracingProcessor, tracing
+
+from galileo.decorator import galileo_context
+from galileo.logger import GalileoLogger
+from galileo.utils.serialization import convert_to_string_dict, serialize_to_str
+
+_logger = logging.getLogger("galileo.handlers.agents")
+
+
+def _get_span_name(span: tracing.Span[Any]) -> str:
+    """Get name for a given OpenAI Agent span."""
+    if name := getattr(span.span_data, "name", None):
```

Comment:
```suggestion
    if name := getattr(span.span_data, "name"):
```
`None` is already default value